app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RegistrationsController < ApplicationController
+
+  def create
+
+    @user = User.new(sign_up_params)
+    @user.save
+    redirect_to challenges_path
+    
+  end
```

Comment:
We generally do not write our own user creation methods if we're using Devise. Check out [Devise documentation on adding sign up params](https://github.com/plataformatec/devise/wiki/How-To:-Allow-users-to-sign-in-using-their-username-or-email-address) (Note: you can leave out the part regarding logging in with username).